datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/utils/filterSchemaRows.ts

```diff
@@ -63,7 +75,9 @@ export function filterSchemaRows(
         if (
             matchesFieldName(fieldName, formattedFilterText) ||
             matchesEditableTagsOrTerms(row, filteredFieldPathsByEditableMetadata) ||
-            matchesTagsOrTerms(row, formattedFilterText, entityRegistry)
+            matchesEditableDescription(row, filteredFieldPathsByEditableMetadata) ||
+            matchesTagsOrTermsOrDescription(row, formattedFilterText, entityRegistry) || // non-editable tags and terms
+            matchesDescription(row.description, formattedFilterText) // non-editable description
```

Comment:
same thing here, we can just check `matchesFieldName`, `matchesEditableTagsOrTermsOrDescription`, and `matchesTagsOrTermsOrDescription`